src/client/Microsoft.Identity.Client/ApiConfig/WindowsBrokerOptions.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.Core;
+using Microsoft.Identity.Client.Internal.Logger;
+using Microsoft.Identity.Client.PlatformsCommon.Factories;
+
+namespace Microsoft.Identity.Client
+{
+
+    /// <summary>
+    /// Advanced options for using the Windows 10 broker.
+    /// For more details see https://aka.ms/msal-net-wam
+    /// </summary>
+#if !SUPPORTS_BROKER || __MOBILE__
```

Comment:
Does `__MOBILE__` flag needs to be added to props?

Reply:
No, it's defined by .NET